keylime/cloud_verifier_common.py

```diff
@@ -290,12 +290,12 @@ def prepare_error(agent, msgtype="revocation", event=None):
     return tosend
 
 
-def validate_agent_data(agent_data):
+def validate_allowlist_data(agent_data):
```

Comment:
Rename to `validate_ima_policy_data`?

We can add more sanity checks then in the future, because currently it only validates the regexes for the exclude list.

Reply:
Done!